R/model.R

```diff
@@ -578,7 +578,7 @@ check_syntax_method <- function(quiet = FALSE,
     } else {
       include_paths_flag <- " --include_paths="
     }
-    stancflags_val <- paste0(stancflags_val, include_paths_flag, include_paths, " ")
+    stancflags_val <- trimws(paste0(stancflags_val, include_paths_flag, include_paths, " "))
```

Comment:
```suggestion
    stancflags_val <- trimws(paste0(include_paths_flag, include_paths, " "))
```

Reply:
stancflags_val is always going to be "" so it can be removed inside paste0. Also change `stancflags_val <- ""` to `stancflags_val <- NULL`. That makes `c(self$stan_file(), stanc_built_options, stancflags_val)` the same as `c(self$stan_file(), stanc_built_options)` if no include flags are used.